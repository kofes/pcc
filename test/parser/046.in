a[2].c[3,i mod 4]
