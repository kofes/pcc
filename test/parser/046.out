   3
      4
   MOD
      I
[
      C
   .
         2
      [
         A

